Tina beta launch post (#1006)
tinacms/tinacms.org: content/blog/tina-is-in-beta.md

## Improving our Tina Starter

The Tina Starter was built originally to show "the power of Tina" while it did that, we didn't feel that it showed a real-world example. So we went back to the drawing board and created our new [Tina Starter](/guides/tina-cloud/starter/overview/), which includes a landing page, blog, and about pages. You can edit and rearrange the content and we styled it with TailwindCSS to give it some extra shine! Below is an example of just some of the work you can do:

![Quickstart Example](/img/edit-alongside-content.gif)

## Media Manager

Media manager was one of the most important features that we needed for Tina Cloud. Our Cloudinary Media manager allows you to change images, upload new images, and delete ones you no longer need without ever leaving the Tina editing experience. 

I wrote a [blog post announcing it](/blog/manage-your-media-with-cloudinary/) and how to implement it into your application.

## Caching Improvements

> Speed and performance have been something we have been actively working on. We introduced some improvements behind the scenes to improve the way we retrieve the data for your site. Tina was already performance first but now it's even better! 

Reviewer: ```suggestion
Speed and performance have been something we have been actively working on. We introduced some improvements behind the scenes to improve the way we retrieve the data for your site. Tina is carefully built with performance in mind and is now faster! 
```

## Creating `@tinacms/toolkit`

TinaCMS was built with small modular packages, this meant that we relied heavily on React context. The dependency mismatches from over-modularizing our toolkit,  led to many bugs related to missing context.

Our open-source team created @tinacms/toolkit which incorporates the essentials of Tina all in one place. This simplifies everything for you as a user and Tina as a product.

You can read about all the updates and why decided to make the changes in our pinned [GitHub issue](https://github.com/tinacms/tinacms/issues/1898).

> ## Vercel Integration

Reviewer: Vercel integration is currently available when you add our starter, we're already working on providing a Vercel deploy button for any app added to Tina Cloud.

We wanted to reduce the friction to almost zero when testing TinaCMS, so we worked on adding Vercel integration. This means if you sign up for an account, you can one-click and deploy in minutes and start playing around with TinaCMS and Tina Cloud, using our Starter. 

## Dashboard Overhaul

> When using Tina Cloud in alpha our dashboard UX wasn't a first-class experience and at times could be confusing. We completely overhauled the dashboard, making it easier and quicker to add an application to the Cloud, invite users, and find the important information such as site URL(s) or client-id.

Reviewer: ```suggestion
When using Tina Cloud in alpha our dashboard UX wasn't a first-class experience and at times could be confusing. We completely overhauled the dashboard, making it easier and quicker to add an application to the Cloud, invite users, and find the important information such as site URL(s) or Client ID.
```

> If you did use the alpha you will need to sign up again as we made large changes to the way we handle user signup in our backend. 

Reviewer: link to migration page?

---
title: Tina is in Beta

> date: '2021-07-27T20:00:00-04:00'

Reviewer 2: Maybe we should change the date to today's date?

Author: Yup it's on the todo, still editing content

author: James Perkins

> last_edited: '2021-07-27T15:39:06.367Z'

Reviewer 2: Same with last_edited

We also removed old guides that no longer promote Tina's best practices and moved some of our other guides into our experimental section. Experimental to us means that we can't guarantee that there won't be bugs or issues with the packages used. 

## Improving our Tina Starter

The Tina Starter was built originally to show "the power of Tina" while it did that, we didn't feel that it showed a real-world example. So we went back to the drawing board and created our new [Tina Starter](/guides/tina-cloud/starter/overview/), which includes a landing page, blog, and about pages. You can edit and rearrange the content and we styled it with TailwindCSS to give it some extra shine! Below is an example of just some of the work you can do:

![Quickstart Example](/img/edit-alongside-content.gif)

## Media Manager

Media manager was one of the most important features that we needed for Tina Cloud. Our Cloudinary Media manager allows you to change images, upload new images, and delete ones you no longer need without ever leaving the Tina editing experience. 

> I wrote a [blog post announcing it](/blog/manage-your-media-with-cloudinary/) and how to implement it into your application.

Reviewer 2: Maybe we could also link to the docs?

## Dashboard Overhaul

When using Tina Cloud in alpha our dashboard UX wasn't a first-class experience and at times could be confusing. We completely overhauled the dashboard, making it easier and quicker to add an application to the Cloud, invite users, and find important information such as site URL(s) or Client ID.

If you did use the alpha you will need to sign up again as we made large changes to the way we handle user signup in our backend. 

## Changes to Content Modeling 

Content Modeling is the core of how you interact with TinaCMS and also retrieve your content. We decided to make an important change and be more primitive based in nature. 

This allows for simplistic queries that don't require disambiguation, we believe this will allow you as a developer to craft queries with ease. 

If you used the alpha of Tina you might want to read this article the team put together to explain all of the [changes and how to migrate](/docs/tina-cloud/migration-overview/). 

## Give Us Feedback!

> The whole team is truly excited to enter the beta phase and hope you will check it out and give us honest feedback. We want to hear about your projects or, let us know how Tina Cloud can help your team make progress.

Reviewer 2: Maybe somewhere in here we should ask folks to join our discord server?

Author: done

### A New Initialize Command in the CLI

> Tina init is my favorite addition to the Tina experience. A single command can bootstrap Tina on a Next.js application and do all the heavy lifting for you. The team spent quite a bit of time working on this, and refining it, to get it just right.The command `npx @tinacms/cli init` ommand currently does the following:

Reviewer: ```suggestion
Tina init is my favorite addition to the Tina experience. A single command can bootstrap Tina on a Next.js application and do all the heavy lifting for you. The team spent quite a bit of time working on this, and refining it, to get it just right. The command `npx @tinacms/cli init` command currently does the following:
```
